spec/requests/api/logging_spec.rb

```diff
@@ -0,0 +1,51 @@
+#
+# REST API Logging Tests
+#
+describe ApiController do
+  describe "Successful Requests logging" do
+    LOG_REQUESTS = {
+      "API Request"    => nil,
+      "Authentication" => nil,
+      "Authorization"  => nil,
+      "Request"        => nil,
+      "Parameters"     => nil,
+      "Response"       => nil
+    }.freeze
+
+    def expect_log_requests(expectations)
+      expectations.each do |category, expectation|
+        expect_any_instance_of(ApiController).to receive(:log_request)
```

Comment:
Wondering if the expectation could be placed on `$api_log` to avoid this use of `expect_any_instance_of` here?